primefaces/src/main/java/org/primefaces/util/ExcelStylesManager.java

```diff
@@ -251,6 +338,51 @@ private CellStyle createCurrencyStyle() {
         return style;
     }
 
+    private CellStyle createDateStyle() {
+        CellStyle style = createDefaultCellStyle();
+        short dataFormat;
+        if (options.getExcelDateFormat() != null) {
+            String format = options.getExcelDateFormat();
+            dataFormat = wb.getCreationHelper().createDataFormat().getFormat(format);
+        }
+        else {
+            dataFormat = 14;
+        }
+        style.setDataFormat(dataFormat);
+        style.setAlignment(HorizontalAlignment.RIGHT);
+        return style;
+    }
+
+    private CellStyle createDateTimeStyle() {
+        CellStyle style = createDefaultCellStyle();
+        short dataFormat;
+        if (options.getExcelDateTimeFormat() != null) {
+            String format = options.getExcelDateTimeFormat();
+            dataFormat = wb.getCreationHelper().createDataFormat().getFormat(format);
+        }
+        else {
+            dataFormat = 22;
```

Comment:
Same here we should Javadoc where these constants came from?

Reply:
I'm going to change the code to clarify where those values come from.